patches/chromium/fix_activate_background_material_on_windows.patch

```diff
@@ -33,6 +33,15 @@ index 29829e282edfa8821bd366a9e9a3755d7f3f8643..9feff09aa83eb88460dce786ab2514f0
  }
  
  void HWNDMessageHandler::SetWindowIcons(const gfx::ImageSkia& window_icon,
+@@ -1755,7 +1755,7 @@ void HWNDMessageHandler::OnActivateApp(BOOL active, DWORD thread_id) {
+   if (delegate_->HasNonClientView() && !active &&
+       thread_id != GetCurrentThreadId()) {
+     // Update the native frame if it is rendering the non-client area.
+-    if (HasSystemFrame()) {
++    if (HasSystemFrame() || is_translucent_) {
```

Comment:
```suggestion
+    if (is_translucent_ || HasSystemFrame()) {
```

_(micronit)_ all other things being equal... when testing `A || B || ...` , put the cheapest tests first. 

Checking a class' bool field is cheaper than `HasSystemFrame()` which at a minimum takes three function calls. So by swapping the order, translucent windows can skip those extra function calls :smile_cat: